tests/conftest.py

```diff
@@ -130,7 +130,7 @@ def one_step_pipeline():
 
     @pipeline
     def _pipeline(step_):
-        pass
+        step_()
```

Comment:
For my own understanding: why are these changes needed here?

Reply:
'Calling' the step inside the pipeline usually is necessary to connect its inputs/outputs to other steps, and at the same time also generates the tfx component that we need to run it during a pipeline.

Reply:
See `BaseStep.__call__`. If you forget to do that right now you'd get an error message saying that a step hasn't been called yet, basically reminding you to call that inside the pipeline. Not sure if we'd want to lift that requirement and run steps without any input artifacts automatically if they're passed to a pipeline, @htahir1 what do you think?

Reply:
You mean steps that are disconnected from everything else? I'm not sure there is a requirement for that tbh. For now I'd stray away from adding functionality not asked for